packages/botonic-react/src/webchat/message-list/styles.ts

```diff
@@ -9,6 +9,21 @@ export const ContainerMessage = styled.div`
   word-wrap: break-word;
 `
 
+export const ContainerMessage = styled(BaseContainerMessage)`
+  /* Default spacing for regular messages */
+`
+
+export const SystemContainerMessage = styled(BaseContainerMessage)`
+  /* No margin between consecutive system messages */
+  margin-top: 0;
+
+  /* Add spacing when transitioning between message types */
+  /* ${ContainerMessage} + &,
+  & + ${ContainerMessage} {
+    margin-top: 12px;
+  } */
+`
```

Comment:
I don't understand these styles. The first styles component doesn't add anything, and the second only adds margin-top: 0?